src/workerd/api/actor-state.c++

```diff
@@ -886,10 +887,23 @@ jsg::Ref<Fetcher> DurableObjectFacets::get(jsg::Lock& js,
         id = ioCtx.getActorOrThrow().cloneId();
       }
 
-      auto actorClass = options.$class->getChannel(ioCtx);
+      DurableObjectClass& actorClass = [&]() -> DurableObjectClass& {
+        KJ_SWITCH_ONEOF(options.$class) {
```

Comment:
Nit: The use of `$class` here is obscure enough that a comment here would likely be helpful for future folks looking into this code.

Reply:
OK I added doc comments to `DurableObjectFacets::StartupOptions`.

FWIW `$class` is just an identifier. The $ is needed so that it's not interpreted as a keyword. JSG removes the $ from the JS API.